CTP/README.md

```diff
@@ -271,42 +278,44 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
 
 - **HA Replication**
   - Prepare
-	* Prepare environments (e.g., one account ``controller`` as controller, one account ``ha_repl_instance1`` as a master test instance, one same account name ``ha_repl_instance1`` on the other machines as a slave test instance)
-	* Check out CTP for each account and configure environment variables for CTP ``controller`` and test instances ``ha_repl_instance1`` (both master and slave)
-      ```
-	  # On controller node
-	  JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
-
-	  # On test instance nodes (master and slave)
-	  CTP_HOME (e.g., export CTP_HOME=$HOME/CTP)
-	  JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
-      ```	
-	* **Example** ``ha_repl.conf`` for test instances (master and slave), scenario and test build:
-      ```
-	  # Test instance information:
-	  env.instance1.master.ssh.host=192.168.1.10
-	  env.instance1.master.ssh.user=ha_repl_instance1
-	  env.instance1.slave.ssh.host=192.168.1.11
-	  env.instance1.slave.ssh.user=ha_repl_instance1
-	  env.instance1.cubrid.cubrid_port_id=1137
-	  env.instance1.ha.ha_port_id=59001
-	  env.instance1.broker.BROKER_PORT=35000
-      ```	
-	  
-      ```
-	  # Define the path of test cases used for testing, it should be checked out on controller node
-	  scenario=${HOME}/cubrid-testcases/sql
-      ```
-	NOTE: the scenario must be checked out on ``controller`` account, not on the test instance
-
-      ```
-	  # Define the URL of test build which will be used to test
-	  # If this parameter is not set or commented out, CTP will execute testing without build installation.
-	  cubrid_download_url=http://127.0.0.1/download/CUBRID-10.1.0.6929-b049ba5-Linux.x86_64.sh 
-	  ```
-
-	 Regarding more explanation for parameters setting, please refer to [CTP/conf/ha_repl.conf](conf/ha_repl.conf)	
-	  
+	* Prepare at least two accounts for the multiple test instances (e.g., one account named as ``controller``, another account named as ``ha_repl_instance1``)

```

Comment:
Actually, HA replication test needs at least three accounts: one controller , one master and one slave.